model/src/pyrenew/process/rtperiodicdiff.py

```diff
@@ -188,7 +192,9 @@ def sample(
         )[0]
 
         return RtPeriodicDiffProcessSample(
-            rt=self.broadcaster(jnp.exp(log_rt.flatten()), duration),
+            rt=SampledValue(
```

Comment:
Same as before, should we remove the extra layer of abstraction and just return a named tuple with a `SampledValue`?

Reply:
Same as my last comment, nothing to do here.